Cargo.toml

```diff
@@ -56,4 +56,4 @@ tokio = { version = "1.30.0", default-features = false }
 wkt = "0.12.0"
 
 [patch.crates-io]
-#geozero = { path = "./geozero" }
```

Comment:
git blames me for changing this line in e3f6502634daa2bbf472e56fe26f6137931fa80a. I'm with you that a local build should always use ./geozero. :+1: +1